lib/view_component/base.rb

```diff
@@ -591,7 +591,7 @@ def validate_collection_parameter!(validate_default: false)
         parameter = validate_default ? collection_parameter : provided_collection_parameter
 
         return unless parameter
-        return if initialize_parameter_names.include?(parameter)
+        return if initialize_parameter_names.include?(parameter) || initialize_parameter_names.include?(:kwargs)
```

Comment:
Does this imply that the splatted argument has to have the name `kwargs`?   That is, would this work with `def initialize(**other_args)`?

Reply:
An excellent question to which I do not yet have a conclusive answer. I'll update the tests and report back.
![i have no idea what im doing](https://user-images.githubusercontent.com/50284/201409062-f5e8008d-6d52-4814-92c7-bb02b7ea0e3e.png)

Reply:
Excellent catch! I've updated to check for `:kwrest` in the flattened initialize parameters!

Reply:
Thanks for making that change.